packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -56,8 +56,8 @@ Requires: %{?scl_prefix}rubygem(pulp_ansible_client) >= 0.7
 Requires: %{?scl_prefix}rubygem(pulp_ansible_client) < 0.8
 Requires: %{?scl_prefix}rubygem(pulp_container_client) >= 2.4.0
 Requires: %{?scl_prefix}rubygem(pulp_container_client) < 2.5.0
-Requires: %{?scl_prefix}rubygem(pulp_deb_client) >= 2.10.0
-Requires: %{?scl_prefix}rubygem(pulp_deb_client) < 2.11.0
+Requires: %{?scl_prefix}rubygem(pulp_deb_client) >= 2.11.0
```

Comment:
I am near certain the API bindings have not actually changed from `2.10.0`, should I nevertheless bump this minimum version to the new version, or not?

Reply:
this is generated from https://github.com/Katello/katello/blob/893cc6dd265754e865c1c93a3a6df73cf9a4a736/katello.gemspec#L59, so you'll need to change it there, and then it will be (semi) automatically updated here.

Reply:
So, like this: https://github.com/Katello/katello/pull/9317
What needs to be merged first?